### ISSUE TITLE
Kivy Import Error on Windows

### ISSUE THREAD

**Issue by toomanybrians** [missionpinball/mpf-mc#224]
When running "mpf both -X -v", this happens:
```
Traceback (most recent call last):
  File "c:\python34\lib\multiprocessing\process.py", line 254, in _bootstrap
    self.run()
  File "c:\python34\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "c:\python34\lib\site-packages\mpf\commands\both.py", line 14, in _start_mc
    module.Command(mpf_path, machine_path, args)
  File "c:\python34\lib\site-packages\mpfmc\commands\mc.py", line 157, in __init__
    self.preprocess_config(mpf_config)
  File "c:\python34\lib\site-packages\mpfmc\commands\mc.py", line 188, in preprocess_config
    from kivy.config import Config
  File "c:\python34\lib\site-packages\kivy\config.py", line 287, in <module>
    from kivy import kivy_config_fn
ImportError: cannot import name 'kivy_config_fn'
```

I had the following versions of the following packages installed (partial list):
```
Kivy (1.9.1)
Kivy-Garden (0.1.4)
kivy.deps.glew (0.1.4)
kivy.deps.glew-dev (0.1.4)
kivy.deps.gstreamer (0.1.7)
kivy.deps.gstreamer-dev (0.1.5)
kivy.deps.sdl2 (0.1.12)
kivy.deps.sdl2-dev (0.1.12)
mpf (0.33.0.dev1, z:\git\mpf)
mpf-mc (0.33.0.dev1, z:\git\mpf-mc)
pip (9.0.1)
Pygments (2.1.3)
pypiwin32 (219)
pyserial (3.2.1)
pyserial-asyncio (0.3)
requests (2.10.0)
ruamel.base (1.0.0)
ruamel.yaml (0.10.23)
setuptools (32.1.0)
typing (3.5.2.2)
wheel (0.29.0)
```

I updated several to:
```
Kivy (1.9.1)
Kivy-Garden (0.1.4)
kivy.deps.glew (0.1.7)
kivy.deps.glew-dev (0.1.4)
kivy.deps.gstreamer (0.1.8)
kivy.deps.gstreamer-dev (0.1.5)
kivy.deps.sdl2 (0.1.14)
kivy.deps.sdl2-dev (0.1.14)
mingwpy (0.1.0b3)
mpf (0.33.0.dev1, z:\git\mpf)
mpf-mc (0.33.0.dev1, z:\git\mpf-mc)
pip (9.0.1)
Pygments (2.1.3)
pypiwin32 (219)
pyserial (3.2.1)
pyserial-asyncio (0.3)
requests (2.12.4)
ruamel.base (1.0.0)
ruamel.yaml (0.10.23)
setuptools (32.1.0)
typing (3.5.2.2)
wheel (0.29.0)
```
Now I get the problem on my system too. (I don't believe I got it before.. will investigate.

Also confirmed this happens via the interactive console:

```
>>> from kivy import kivy_config_fn
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'kivy_config_fn'
```

I can import other things from kivy no prob

**Comment by toomanybrians** [comment]
Looks like all the kivy.deps packages were updated today on PyPI.

https://pypi.python.org/pypi/kivy.deps.gstreamer
https://pypi.python.org/pypi/kivy.deps.glew
https://pypi.python.org/pypi/kivy.deps.sdl2

Unfortunately the binaries for the old version of gstreamer are not there, so I can't easily test. I rolled back the other versions and still have the same problem. I'll dig into the Kivy group or see if maybe Kivy itself has been updated.
